WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingCoordinator.swift

```diff
@@ -12,10 +12,11 @@ final class StoreOnboardingCoordinator: Coordinator {
 
     /// Navigates to the fullscreen store onboarding view.
     func start() {
-        let onboardingController = UIHostingController(rootView: StoreOnboardingView(viewModel: .init(isExpanded: true), taskTapped: { [weak self] task in
-            self?.start(task: task)
-        }))
-        navigationController.show(onboardingController, sender: self)
+        let onboardingView = StoreOnboardingView(viewModel: .init(isExpanded: true),
+                                                 taskTapped: { [weak self] task in self?.start(task: task) },
+                                                 dismissAction: { [weak self] in self?.navigationController.dismiss(animated: true) })
+        let onboardingController = UIHostingController(rootView: onboardingView)
+        navigationController.present(onboardingController, animated: true)
```

Comment:
💡 If we want to display any navigation bar item, we should embed the `onboardingController` within a `UINavigationController` and maybe update the navigation bar later with the extension method `configureTransparentNavigationBar`.

Reply:
Done in f3fef5e655139d8a02972da2ef5f2f4b50d31e13